writing-gleam/gleam-toml.md

```diff
@@ -41,6 +41,17 @@ links = [
   { title = "Other site", href = "https://another.example.com" },
 ]
 
+# Modules that should be considered "internal" and will not be included in
+# generated documentation. Note this currently only affects documentation;
+# public types and functions defined in these modules are still public.
+#
+# Items in this list are "globs" that are matched against module names. See:
+# https://docs.rs/glob/latest/glob/struct.Pattern.html
```

Comment:
Fab! Would you mind documenting what the default values are 🙏

Reply:
@lpil Done! I assume you will squash-merge, but if not let me know and I'll fixup the wip commit.